regparser/grammar/amdpar.py

```diff
@@ -206,23 +209,34 @@ def _paren_join(elements):
     intro_text_marker + of_connective
     + atomic.paragraph_marker
     + comment_p
-    ).setParseAction(lambda m: tokens.Paragraph([None,
-        'Interpretations', None, None, m.level2, m.level3,
+    ).setParseAction(lambda m: tokens.Paragraph([
+        None, 'Interpretations', None, None, m.level2, m.level3,
         m.level4], field=tokens.Paragraph.TEXT_FIELD))
 
 single_par = (
     unified.marker_paragraph
     + Optional(intro_text_marker)
-    ).setParseAction(lambda m: tokens.Paragraph([None, None, None,
-        m.p1, m.p2, m.p3, m.p4, m.plaintext_p5, m.plaintext_p6],
+    ).setParseAction(lambda m: tokens.Paragraph([
+        None, None, None, m.p1, m.p2, m.p3, m.p4, m.plaintext_p5,
+        m.plaintext_p6],
         field=(tokens.Paragraph.TEXT_FIELD if m[-1] == 'text' else None)))
 section_single_par = (
     unified.marker_part_section
     + unified.depth1_p
     + Optional(intro_text_marker)
-    ).setParseAction(lambda m: tokens.Paragraph([m.part, None,
-        m.section, m.p1, m.p2, m.p3, m.p4, m.plaintext_p5, m.plaintext_p6],
+    ).setParseAction(lambda m: tokens.Paragraph([
+        m.part, None, m.section, m.p1, m.p2, m.p3, m.p4, m.plaintext_p5,
+        m.plaintext_p6],
         field=(tokens.Paragraph.TEXT_FIELD if m[-1] == 'text' else None)))
+single_par_section = (
```

Comment:
I absolutely agree! The tests are usually fairly descriptive, but they're all the way in another file.